tests/integration/test_s3_zero_copy_replication/test.py

```diff
@@ -60,6 +60,8 @@ def wait_for_active_parts(node, num_expected_parts, table_name, timeout=30):
     assert num_parts == num_expected_parts
 
 
+# Result of `get_large_objects_count` can be changed in other tests, so run this case at the beginning
+@pytest.mark.order('first')
```

Comment:
Stop, seriously, they define order with keywords `'first'`, `'second'` and so on!?!?! Maybe it's possible to use numbers `0` or `1`?

Reply:
Yes :) https://github.com/pytest-dev/pytest-order/blob/1e873d5c2b67211ed3bc578ee72bb3d986906532/pytest_order/sorter.py#L30-L47

We can replace it with `0`

Reply:
```suggestion
@pytest.mark.order(0)
```

Reply:
Looks like Python get the best practices from Perl :)